ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,96 @@
+base_images:
+  ansible:
+    name: "4.11"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.11"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.11"
+  stable:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.11"
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.11"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aro-p1-f30
+  cron: 59 13 20 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.10 and @azure-ipi and not @fips'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.10 and @azure-ipi and not @fips'
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE410},${RELEASE_IMAGE_STABLE},${RELEASE_IMAGE_TARGET}"
```

Comment:
Continue the last discussion in https://github.com/openshift/release/pull/34479#discussion_r1116477873, @mgahagan73 you mentioned:
```
I think it was upgrading to latest stable 4.10 -> latest stable 4.11 -> latest nightly 4.11
```
I am wondering how to decide the source version for the ARO cluster?

Reply:
according to the aro team the default version is hardcoded in the RP here: https://github.com/Azure/ARO-RP/blob/master/pkg/util/version/const.go#L29-L32
Going forward (prob q3 work) we might decouple it from the RP and instead stored in a database or something along the lines.

future az cli versions will have an updated aro extension what adds a get-version option that will display (and at some point allow you to select) the version.

Reply:
For now I was thinking that once ARO moves to 4.11 (likely 4.11.26 or newer) we could just remove the RELEASE_IMAGE_INTERMEDIATE410 from set-upgrade-releases, then when we get to 4.12 just remove the 4.11 job config entirely.

Reply:
It sounds a plan to drop the out-of-date job config entirely once the source ARO version get updated.